.github/workflows/test.yml

```diff
@@ -0,0 +1,87 @@
+name: Quality Assurance
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-linux:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        platform: [x32, x64]
```

Comment:
Currently, your build will run 4 variants (x32 gcc, x32 clang, x64 gcc, x64 clang) but all builds will be identical because you don't make use of `platform` or `compiler`. Comment `platform` and `compiler` until you actually use them.

Reply:
This one's still open.

Reply:
Yes, I'm still looking for the solution. ModSecurity [uses] (almost) same [scheme](https://github.com/owasp-modsecurity/ModSecurity/blob/v3/master/.github/workflows/ci.yml#L11-L14), I tried to copy from there.

Reply:
Okay, I give up. There is no any difference between platform and compilers - see the [last log](https://github.com/owasp-modsecurity/ModSecurity-nginx/actions/runs/8679760810/job/23799132568) when I tried. Removed those options, we run tests only x64 with gcc.